toolchain/win/BUILD.gn

```diff
@@ -171,24 +171,41 @@ template("msvc_toolchain") {
       outputs = [
         "$object_subdir/{{source_name_part}}.obj",
       ]
-
-      command = "$env_wrapper$cl /nologo /showIncludes ${clflags} $sys_include_flags{{defines}} {{include_dirs}} {{cflags}} {{cflags_c}} /c {{source}} /Fo{{output}} /Fd\"$pdbname\""
+      if (target_name == "uwp_clang_arm" || target_name == "win_clang_arm") {
```

Comment:
Please make it an rspFile for all paths.

Reply:
Actually, not sure why one path has rspFile and others do not. Original does not have rspPath.